src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -36,27 +39,42 @@ export const MyProfile = () => {
     queryFn: () => fetchPerson(personId),
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_person'), variant: 'error' })),
   });
-  const person = personQuery.data;
 
-  const hasActiveEmployment = filterActiveAffiliations(person?.affiliations).length > 0;
+  const person = personQuery.data;
 
   const firstName = getValueByKey('FirstName', person?.names);
   const lastName = getValueByKey('LastName', person?.names);
   const personPreferredFirstName = getValueByKey('PreferredFirstName', person?.names);
   const personPreferredLastName = getValueByKey('PreferredLastName', person?.names);
-  const [editPreferredNames, setEditPreferredNames] = useState(false);
   const personTelephone = person?.contactDetails?.telephone;
+  const [editPreferredNames, setEditPreferredNames] = useState(false);
+
+  const typographyStyling = {
+    height: '100%',
+    display: 'flex',
+    alignItems: 'center',
+  };
 
   const initialValues: CristinPersonFormData = {
     preferredFirstName: personPreferredFirstName ? personPreferredFirstName : firstName,
     preferredLastName: personPreferredLastName ? personPreferredLastName : lastName,
+    contactDetails: {
+      telephone: personTelephone ? personTelephone : '',
+      email: person?.contactDetails?.email ? person.contactDetails.email : '',
+      webPage: person?.contactDetails?.webPage ? person.contactDetails.webPage : '',
+    },
   };
 
   const updatePerson = async (values: CristinPersonFormData) => {
     if (user.cristinId) {
       const payload: CristinPersonFormData = {
         preferredFirstName: values.preferredFirstName === '' ? null : values.preferredFirstName?.trim(),
         preferredLastName: values.preferredLastName === '' ? null : values.preferredLastName?.trim(),
+        contactDetails: {
+          telephone: values.contactDetails?.telephone === '' ? null : values.contactDetails?.telephone?.trim(),
+          email: values.contactDetails?.email === '' ? null : values.contactDetails?.email?.trim(),
+          webPage: values.contactDetails?.webPage === '' ? null : values.contactDetails?.webPage?.trim(),
```

Comment:
Har en hypotese om at dette blir det samme 🤔 
```suggestion
          telephone: values.contactDetails?.telephone?.trim(),
          email: values.contactDetails?.email?.trim(),
          webPage: values.contactDetails?.webPage?.trim(),
```